File: Garnik/CS50/Week1/mario.c
#include <stdio.h>

int main(void){
    int height=6;
    for (int i=1; i<=height; i++){
        for (int n=i; n<height; n++){
            printf(" ");
        }
        for (int n=1; n<=i; n++){
            printf("#");
        }
        printf("    ");
        for (int n=1; n<=i; n++){
            printf("#");
        }
        
        printf("\n");
    }
    
    
}

File: Harut/workspace/pset6/mario/more/mario.py
n = 0
while True:
    n = input("height: ")
    if n.isdigit() and int(n) > 0 and int(n) < 9:
        break
n = int(n)

for i in range(1, n+1):
    print(' ' * (n-i) + '#' * i + '  ' + '#' * i)